### DISCUSSION TITLE
Exporting one text column with each row in an individual text file

### DISCUSSION THREAD

**Discussion by rjalexa** [dbeaver/dbeaver#15056] [Q&A]
Dear friends,
have a DB with a table containing a column with each row (a few hundred) being a long text (5-8000 chars usually).

I need to export each text to an individual text file with an arbitrary (sequential for example) but unique filename like 00001.txt, 00002.txt.....00157.txt etc.

I run the very simple query to generate the results as follows:
select
	scraped_text
from
	atti
where 
	category = 'AGENZIA ITALIANA DEL FARMACO' 

and obtain all the rows I need. I then select all the resulting rows and right click to get Export data ... there I select TXT, extraction type single query and selected rows only,  no changes in formatting settings,  in output select the proper directory, 

Is there any way to get one file for each row? Thank you

**Comment by ShadelessFox** [comment]
Hello @rjalexa,

You can set <kbd>Split output file</kbd> option, located at the <kbd>Output</kbd> page, to `1`.

**Comment by rjalexa** [comment]
Thank I followed your suggestion but still I get one single file for all records.
![image](https://user-images.githubusercontent.com/15216911/147569294-349c0d39-8f1b-4fa1-a23f-1d34a3336392.png)
